build/configs/rtl8721csm/rtl8721csm_download.sh

```diff
@@ -204,21 +89,13 @@ function get_executable_name()
 ##Utility function to get partition index ##
 function get_partition_index()
 {
-	case $1 in
-		km0_bl | Km0_bl | KM0_BL) echo "0";;
-		km4_bl | Km4_bl | KM4_BL) echo "1";;
-		kernel | Kernel | KERNEL) echo "2";;
-		userfs | Userfs | USERFS)
-		for i in "${!parts[@]}"
-		do
-		   if [[ "${parts[$i]}" = "userfs" ]]; then
-			echo $i
-		   fi
-		done
-		;;
-		*) echo "No Matching Partition"
-		exit 1
-	esac
+	for idx in ${!parts[@]}; do
+		if [[ ${parts[$idx],,} == ${1,,} ]]; then
+			echo $idx
+			exit 1
+		fi
+	done
+	echo -1
```

Comment:
Do you have any reason to print the `-1`?
If no, please remove this line.

Reply:
It is for unregistered commands like `make download abcd`